external/src/main/java/com/redhat/cloud/policies/engine/actions/plugins/NotificationActionPluginListener.java

```diff
@@ -44,72 +47,107 @@
     public static final String APP_NAME = "policies";
     public static final String EVENT_TYPE_NAME = "policy-triggered";
 
+    private final Logger log = Logger.getLogger(this.getClass().getName());
+    private final ConcurrentSkipListMap<String, PoliciesAction> notifyBuffer = new ConcurrentSkipListMap<>();
 
     @Inject
     @Channel("webhook")
     @OnOverflow(OnOverflow.Strategy.UNBOUNDED_BUFFER)
     Emitter<String> channel;
 
     @Inject
-    @Metric(absolute = true, name = "engine.actions.webhook.processed")
+    @Metric(absolute = true, name = "engine.actions.notifications.processed")
```

Comment:
@pilhuhn Do you know if is OK to update these or if something breaks?

Reply:
It will probably break the metric monitoring in Prometheus but metrics can be renamed so it would only be a temporary issue.

Reply:
@spinder Do you manage the grafana boards? This would need an update on ci/qa after we merge this one. And there is a new one below (`engine.actions.notifications.aggregated`)